src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```
@@ -7,7 +7,7 @@ services:
     arguments:
       - '@prestashop.adapter.product.image.uploader.product_image_uploader'
       - '@prestashop.adapter.product.image.repository.product_image_multishop_repository'
-      - "@prestashop.adapter.image.image_validator"
+      - "@prestashop.adapter.image.product_image_validator"
```

```suggestion
      - "@prestashop.adapter.image.image_validator"
```
Improving the existing validator should be enough